components/popover/stories/template.js

```diff
@@ -71,13 +71,25 @@ export const Template = ({
 					}
 					if (position.startsWith("top")) {
 						y = rect.top - popHeight;
-						yOffset = withTip
-							? "- (var(--spectrum-popover-pointer-height) + var(--spectrum-popover-animation-distance) - 1px)"
-							: "- var(--spectrum-popover-animation-distance)";
+						if (isOpen) {
+							yOffset = withTip
+								? "- (var(--spectrum-popover-pointer-height) - 1px)"
+								: "";
+						}
```

Comment:
I've dropped this animation fix unfortunately, because it no longer worked after rebase and because we have a followup issue to refactor the positioning.